.github/workflows/unit_tests.yml

```diff
@@ -158,13 +158,13 @@ jobs:
     - name: Install MNE (stable)
       if: "matrix.mne-version == 'mne-stable'"
       run: |
-        git clone --depth 1 https://github.com/mne-tools/mne-python.git -b maint/1.0
+        git clone --depth 1 --single-branch --branch maint/1.0 https://github.com/mne-tools/mne-python.git -b maint/1.0
         python -m pip install -e ./mne-python
 
     - name: Install MNE (main)
       if: "matrix.mne-version == 'mne-main'"
       run: |
-        git clone --depth 1 https://github.com/mne-tools/mne-python.git -b main
+        git clone --depth 1 --single-branch --branch main https://github.com/mne-tools/mne-python.git -b main
```

Comment:
shall we keep this?

Reply:
Yes it's just more specific/explicit and a tiny bit faster

Reply:
but we should probably remove the trailing `-b main`, right? that's just a single flag specified twice?